api/v1/db/oracledb/game-dao.js

```diff
@@ -177,6 +179,37 @@ const cleanTableCardsByGameId = async (gameId, connection) => {
   await connection.execute(cleanCardSqlQuery, sqlParams);
 };
 
+const insertCardsByGameId = async (gameId, tableCards, connection) => {
+  // const flattenedArray = _.flatten(_.map(tableCards, card => (_.values(card))));
+  const flattenedArray = _.flatten(_.map(tableCards,
+    card => (_.values(_(card).toPairs().sortBy(0).fromPairs()
+      .value()))));
```

Comment:
It looks like extra work is being done here since the card is converted to pairs of arrays, sorted, and converted back. You could probably simplify this by doing something like
```js
const newArr = _.reduce(arr, (result, item) => {
  result.push(item.a);
  result.push(item.b);
  return result;
}, []);
```
Could you look into this?